packages/db-objects-redis/src/lib/objects/objectsInRedisClient.ts

```diff
@@ -4202,44 +4196,44 @@ export class ObjectsInRedisClient {
     }
 
     // no callback provided, thus we return a result Promise
-    getObjectView(
-        design: string,
-        search: string,
+    getObjectView<Design extends string = string, Search extends string = string>(
```

Comment:
I think these are okay for now, but we should look into not having to duplicate the definitions from `@types/iobroker`.